packages/opentelemetry-core/src/utils/environment.ts

```diff
@@ -118,30 +184,34 @@ function setLogLevelFromEnv(
  * Parses environment values
  * @param values
  */
-export function parseEnvironment(values: ENVIRONMENT_MAP): ENVIRONMENT {
-  const environment: ENVIRONMENT_MAP = {};
+export function parseEnvironment(values: RAW_ENVIRONMENT): ENVIRONMENT {
+  const result: ENVIRONMENT = {};
```

Comment:
why did you change the param name which is typeof `ENVIRONMENT` to be `result` instead of `environment` ?

Reply:
I answered on the later comment

Reply:
I'm asking about variable name, it was `environment`, you changed to be `result`.
What's the benefit to have this
```
const result: ENVIRONMENT = {};
```
instead of  this
```
const environment: ENVIRONMENT = {};
```

Reply:
I think it makes the distinction between raw and parsed environment objects slightly better, `environment` could be both. But let me revert that to make PR review simpler.